src/main/webapp/assets/modules/salary-report/salary-review.con.js

```diff
@@ -99,12 +105,7 @@ techlooper.controller("salaryReviewController", function ($location, $scope, vnw
       catch (e) {}
     }
 
-    if (campaign.id) {
-      $http.get(jsonValue.httpUri.salaryReview + "/" + campaign.id)
-        .success(function (data, status, headers, config) {
-          $scope.salaryReview = data;
-          //$scope.salaryReview.hasCam = !$.isEmptyObject(campaign);
-          utils.sendNotification(jsonValue.notifications.loaded);
+    if (campaign.id) nValue.notifications.loaded);
```

Comment:
Expected '{' and instead saw 'nValue'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>'nValue' is not defined.